src/rust/dbv2/README.md

```diff
@@ -0,0 +1,28 @@
+# Database
+
+This repository provides migrations to create the Econia event database, and Rust types to work with the event database.
+
+The aggregator furthermore adds tables to this database under the `aggregator` schema. Those are disucssed in the `src/rust/aggregator` crate.
+
+# Contributing
+
+When creating a new migration, before committing, please make sure to use [sql-formatter](https://github.com/sql-formatter-org/sql-formatter) with the provided configuration.
+
+Also make sure to add the corresponding documentation to `models.md`.
+
+# Documentation
+
+## diesel
```

Comment:
If it is going to be lowercase in a header, how about monospace too?

```
## `diesel`
```

Reply:
Addressed in c7683cb9.